api/app/hfi/hfi_calc.py

```diff
@@ -78,8 +75,8 @@ def generate_station_daily(raw_daily,  # pylint: disable=too-many-locals
 
     return StationDaily(
         code=station.code,
-        date=timestamp,  # TODO: passing a timestamp into a date object!
-        status=raw_daily.get('recordType', '').get('id', None),
+        date=datetime.fromtimestamp(raw_daily['weatherTimestamp'] / 1000, tz=timezone.utc),
```

Comment:
weatherTimestamp can never be None, so not using .get, rather using index so it will fail if it's not there. Also moved extraction down here.